visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.scss

```diff
@@ -153,10 +153,6 @@ cc-ribbon-bar {
 		}
 	}
 
-	.mat-card:not([class*="mat-elevation-z"]) {
```

Comment:
Thanks for removing this unsed statement. I think we actually don't need it anymore.